CHANGELOG.md

```diff
@@ -48,7 +48,7 @@ This release contains several **minor breaking changes**. Please review your cod
 * `Verify[All]` fails because of lazy (instead of eager) setup argument expression evaluation (@aeslinger, #711)
 * `ArgumentOutOfRangeException` when setup expression contains indexer access (@mosentok, #714)
 * Incorrect implementation of `Times.Equals` (@stakx, #805)
-
+* Calling `ToString()` on `SetupPhrase` does not return expression (@jacob-ewald, #810)
```

Comment:
This is somewhat misplaced as it is an enhancement (*Added*), not a bug fix (*Fixed*): The ability to inspect a setup while it's being built was never specified.

What's even more important though is that the audience (users of the Moq library) typically won't have a clue what `SetupPhrase` is, nor should they care: It's an implementation detail that's never seen outside the library.

I'd prefer it&mdash;but only if it doesn't make you feel cheated out of your contribution being formally acknowledged&mdash; if we didn't list this change in the changelog at all. It probably doesn't clear the bar for things that users typically need to know about, as the use case is somewhat unusual. (In typical usage, you wouldn't log your setups to the console. You shouldn't even need to store them in a variable! Typically, verification error messages should mention any setups relevant to finding out what's wrong, so the use case we're dealing with here is somewhat unusual to begin with.)

If you'd really like to see this in the changelog despite these counterarguments, then please (a) move it to the *Added* section; (b) make it sound like an addition, not a bug fix, and (c) rephrase it such that it doesn't mention some internal implementation classes.

Reply:
I'm not concerned about my name showing up in the changelog, I was just following the contribution guidelines. This change is [reverted](https://github.com/moq/moq4/pull/812/commits/b78daf95a1e673651e96bcde6878f622c45c23fc).